nmosregistration/common/routes.py

```diff
@@ -164,13 +168,14 @@ def _delete(self, resource_type, resource_id):
 
     @route('/')
     def __versionroot(self):
-        return [ 'resource/', 'health/' ]
+        return ['resource/', 'health/']
 
     @route('/resource', methods=['GET', 'POST'], auto_json=False)
+    @RequiresAuth()
```

Comment:
I'd have preferred this in a separate commit to the linting, but don't worry in this case.